### ISSUE TITLE
INVALID_ARGUMENT: Received a label value of 66908 which is outside the valid range of [0, 66908)

### ISSUE THREAD

**Issue by Leizhengwang** [biarne-a/hm-two-step-reco#3]
File "\hm-two-step-reco-main\two-tower-cg\custom_cross_entropy_loss.py", line 22, in __call__
      loss = tf.nn.sparse_softmax_cross_entropy_with_logits(labels=true_labels, logits=logits)
Node: 'SparseSoftmaxCrossEntropyWithLogits/SparseSoftmaxCrossEntropyWithLogits'
Received a label value of 66908 which is outside the valid range of [0, 66908).  Label values: 49662 50073 50073 55641 53538 66908 
	 [[{{node SparseSoftmaxCrossEntropyWithLogits/SparseSoftmaxCrossEntropyWithLogits}}]] [Op:__inference_test_function_83141]
